src/app/components/pages/settings/pending-transactions/pending-transactions.component.ts

```diff
@@ -37,13 +38,16 @@ export class PendingTransactionsComponent implements OnInit, OnDestroy {
   }
 
   private loadTransactions(value: number) {
+    this.isLoading = true;
+
     const showAllTransactions = value === DoubleButtonActive.RightButton;
     this.walletService.getAllPendingTransactions()
```

Comment:
Please, delay the execution of this 32ms. The reason is that if the server responds very quickly, pressing the "Mine" and "All" buttons will not show the loading animation, so the user might think that the wallet did nothing and that it is working incorrectly.